pkgs/applications/misc/barrier/default.nix

```diff
@@ -1,26 +1,20 @@
-{ stdenv, fetchpatch, fetchurl, cmake, curl, xorg, avahi, qt5,
+{ stdenv, fetchpatch, fetchurl, cmake, curl, xorg, avahi, qtbase, mkDerivation,
   avahiWithLibdnssdCompat ? avahi.override { withLibdnssdCompat = true; }
 }:
 
-stdenv.mkDerivation rec {
+mkDerivation rec {
   name = "barrier-${version}";
-  version = "2.1.1";
+  version = "2.2.0";
```

Comment:
v2.3.0
https://github.com/debauchee/barrier/releases/tag/v2.3.0

Reply:
i decided to go 2.2.0 because 2.3.0 isn't marked as stable yet, but do you think it's worth upgrading to 2.3.0 now?

Reply:
Not sure if this is true anymore
> I'll mark this as pre-release first, and people who feel confident enough can test it. The main thing that worries me is the new macOS release, there hasn't been one in a while. As I get feedback and feel confident enough, I'll eventually mark this as just a release.

GitHub seems to be calling it the latest.

Reply:
Perhaps it we support `macOS 10.12` (can't remember).

But I'd go with your intuition first since you actually use this.

Reply:
well i launched it and everything seems to be working, so I'll make it 2.3.0 for now but may revert later if bugs arise

Reply:
sounds good, it's probably not likely then since they made it a formal release.